apps/elasticsearch/templates/services/app/dc.yml.j2

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: elasticsearch
+    service: elasticsearch
```

Comment:
```suggestion
    service: app
```

Reply:
I think it is not homogeneous across our apps :cry:

Reply:
Maybe I should rename the service directory to `elasticsearch` instead of `app`, and keep labels as is ?
I don't like the name `app` but I lacked inspiration :smile: 

And if in a future we add services like kibana or other elastic related services, it will make more sense.

What do you think about it ?

Reply:
I would let it as is, and make a separated PR, that homogenize this for all apps. Can you declare a PR for this @madmatah?

Reply:
We use `app` to avoid repetition with the application name but honestly naming it with its real name is a better idea IMO. Is there any limitation we don't know @jmaupetit ?

Reply:
No limitations just a convention, see my comment above.

Reply:
Thanks for the explanations.
I'll fix this one (because it's fast and i prefer to introduce a cleaner application) and open an issue for the rest of the apps :+1: